plugins/fireeye_hx/plugin.spec.yaml

```diff
@@ -314,11 +315,20 @@ connection:
     title: Username and Password
     description: Username and password to authenticate with FireEye HX
     required: true
+    example: '{"username": "user", "password": "password"}'
   url:
     type: string
     title: URL
     description: URL to the appliance, e.g. https://test.fireeye.com
     required: true
+    example: "https://example.com"
+  ssl_verify:
+    title: SSL Verify
+    description: Validate SSL certificate
+    type: boolean
+    default: true
+    required: false
```

Comment:
IMO a boolean value shouldn't be optional - it's either true or false. I would make this required and default it to `true`.